### PR TITLE
Add Inertia Icon

### LINKED ISSUE #7371: Request: InertiaJS
### Brand Name

InertiaJS

### Website

https://inertiajs.com/

### Popularity Metric

The similar webrank is currently : 203,006 (233k visits)

I'm an inertiajs enthusiast so i think the future is bright for this project, i love working with it so i'm biased on this.

### Official Resources for Icon and Color

Color : #8d5aea

Svg (from official website, do not contain only the brand logo ... 

```
<svg class="block fill-current text-white" style="height:25px" viewBox="0 0 275.3 50.5"><path d="M231.2 16.1h-17.8l17.2 17.2-17.2 17.2h17.8l17.2-17.2z"></path><path d="M258.1 16.1h-17.8l17.2 17.2-17.2 17.2h17.8l17.2-17.2z"></path><path d="M6 15.3h10.3l-6 34.2H0l6-34.2zm.6-9.1C7.2 2.9 10.3 0 13.7 0s5.7 2.8 5.2 6.2c-.5 3.4-3.7 6.2-7.2 6.2s-5.6-3-5.1-6.2zM54.3 28.5l-3.7 21H40.4L43.8 30c.8-4.4-1.6-6.2-4.9-6.2-3.4 0-6.5 2-7.5 6.6L28 49.5H17.8l6-34.2h10.3l-.5 3.2c2.3-2.6 6.2-4.2 10.1-4.2 6.9.1 12.2 5.1 10.6 14.2zM94.5 32.4c-.1.8-.5 2.7-1.1 4.1H68.9c.6 3.8 3.8 4.8 7 4.8 2.9 0 5.2-.8 7.2-2.7l7.2 5.9c-4 4-8.7 6-15 6-11.8 0-18-8.5-16.3-18.7a20.7 20.7 0 0 1 20.5-17.4c9.8 0 16.9 7.6 15 18zm-9.7-3.7c-.3-3.8-3-5.3-6.2-5.3a8.9 8.9 0 0 0-8.3 5.3h14.5zM123.9 14.6l-2 11.6c-4-.6-10.5.8-11.7 7.8l.1-.4-2.8 15.9H97.3l6-34.2h10.3l-1.1 6.2c2.1-4.7 6.6-6.9 11.4-6.9zM137.8 37.3c-.5 3.1 2 3.3 6.6 2.9l-1.6 9.3c-12.3 1.4-16.9-2.7-15.2-12.2l2.1-12.1h-5.5l1.8-9.9h5.4l1.2-6.5 10.8-3.1-1.7 9.6h7.1l-1.8 9.9h-7l-2.2 12.1zM155.3 15.3h10.3l-6 34.2h-10.3l6-34.2zm.6-9.1c.5-3.3 3.7-6.2 7.1-6.2s5.7 2.8 5.2 6.2c-.5 3.4-3.7 6.2-7.2 6.2s-5.7-3-5.1-6.2zM208.1 15.3l-6 34.2h-10.3l.4-2.3a15.5 15.5 0 0 1-10.3 3.3c-11.1 0-15.3-9.6-13.5-18.9 1.6-8.8 8.6-17.2 19.2-17.2 4.5 0 7.7 1.8 9.6 4.6l.6-3.6h10.3zm-13.2 17.2c.9-5.2-1.9-8.4-6.6-8.4a9.5 9.5 0 0 0-9.5 8.3c-.9 5.1 1.8 8.3 6.6 8.3 4.6.1 8.6-3.1 9.5-8.2z"></path></svg>
```

### Additional Comments

Thank you for making these available !

### Contributing

Yes

### LINKED ISSUE #8503: Request icon: Inertia.js
### Brand Name

Inertia

### Website

https://inertiajs.com/

### Popularity Metric

The similarweb rank is 286,911 (global)

### Official Resources for Icon and Color

No SVG found.
Small logo : https://avatars.githubusercontent.com/u/47703742?s=200&v=4
Large : https://raw.githubusercontent.com/inertiajs/.github/master/LOGO.png

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by mchev**
Inertia

**Issue:** closes #7371 

**Similarweb rank:** : 286,911

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by LitoMore** [comment]
Hi @mchev, the SVG one can be found at their homepage. Let's use that one as reference.

File: _data/simple-icons.json
Comment (LitoMore) [inline _data/simple-icons.json:5477]:
```suggestion
            "hex": "000000",
```

Let's use black as its primary color.

**Comment by LitoMore** [comment]
I just leave a message on their Discord for asking the correct primary color. Let's wait for their response.

https://discord.com/channels/592327939920494592/592327939920494594/1093076623265370142

**Comment by mchev** [comment]
> I just leave a message on their Discord for asking the correct primary color. Let's wait for their response.
> 
> https://discord.com/channels/592327939920494592/592327939920494594/1093076623265370142

It looks like the icon itself is white. But why not use the #9553e9 (from gradient) as primary color.

File: _data/simple-icons.json
Comment (LitoMore) [inline _data/simple-icons.json:5477]:
```suggestion
            "hex": "9553E9",
```

@mchev We just got their approve to use `#9553e9` as primary color.

**Comment by ConrDev** [comment]
here i grabbed the official `>>` logo that is issued in their original logo:

https://github.com/simple-icons/simple-icons/issues/8300

**Comment by LitoMore** [comment]
2023-06-02 at 23 52 37@2x

The icon still has some differences. Could you point out which resource are you using as reference? BTW, you could use the resource in https://github.com/simple-icons/simple-icons/pull/8508#issuecomment-1573300245 as a reference.

**Comment by mchev** [comment]
Sorry but i really don't understand what are the issues here. 

Linter check : `collinear-segments Collinear segment "l-6.668 6.668" found at index 174 (should be removed)`
I have no idea what that means.

Sizes : I tried 3 different versions from official sources. If you have time to just share the correct SVG code with correct viewBox in comments i'll push it.

There is also https://github.com/simple-icons/simple-icons/pull/7376 (where i copy/pasted the last attempt)

**Comment by ConrDev** [comment]
> 2023-06-02 at 23 52 37@2x
> 
> 
> 
> The icon still has some differences. Could you point out which resource are you using as reference? BTW, you could use the resource in https://github.com/simple-icons/simple-icons/pull/8508#issuecomment-1573300245 as a reference.

i don't really get why mine was closed for duplicate tbh, when the icon svg i shared is different from the others :p

File: icons/inertia.svg
Comment (PratyushJayachandran) [inline icons/inertia.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Inertia</title><path d="M6.901 5.331H0L6.669 12 0 18.669h6.901L13.571 12 6.9 5.331zm10.43 0H10.43L17.099 12l-6.67 6.669h6.902L24 12l-6.669-6.669z"/></svg>
```

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
The issue should be fixed now

**Review by adamrusted — APPROVED** [review]
LGTM! Icon matches, and we've got the go-ahead to use this brand colour for now! Thanks folks!

**Comment by adamrusted** [comment]
Darn co-linear segments. Hopefully @PratyushJayachandran has got it passing the linter with their improvements!

**Pull request merged**
